pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SerializeJavaR.scala

```diff
@@ -0,0 +1,297 @@
+package edu.berkeley.cs.amplab.sparkr
+
+import scala.collection.JavaConversions._
+
+import java.io.DataInputStream
+import java.io.DataOutputStream
+
+/**
+ * Utility functions to serialize, deserialize objects to / from R
+ */
+object SerializeJavaR {
```

Comment:
Hmm I renamed it to SerDeJVMR - I dont really like the name as there is no nice break between the words JVM and R. The main reason I had chosen Java gives us is with CamelCasing we can figure out the word boundaries, but I guess this is fine for now